apps/dashboard/src/@3rdweb-sdk/react/components/connect-wallet/index.tsx

```diff
@@ -14,6 +14,7 @@ import Link from "next/link";
 import { useCallback, useMemo } from "react";
```

Comment:
Need to check something - On local the Network list was not scrollable when the NetworkSwitcher button is used in Deploy page. Need to check this in vercel preview to confirm